app/search/models.py

```diff
@@ -75,6 +75,17 @@ def contacts(self):
         return contacts
 
 
+    @property
+    def conpStatus(self):
+        conpStatus = None
+        extraprops = self.descriptor.get('extraProperties', {})
+        for prop in extraprops:
+            if prop.get('category') == 'CONP_status':
+                conpStatus = prop.get('values')[0].get('value')
+
+        return conpStatus
```

Comment:
There should be a default value for `conpStatus` or the portal will crash when the DATS model doesn't have this property. Suggestion: `external`.

Reply:
Thanks for the feedback, I've implemented that now.